### DISCUSSION TITLE
Meeting: October 15th, 2024

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#157] [Announcements]
## Weekly meetings happen on Tuesdays

**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Wrap-up and seed the next agenda (5 mins) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by earth2marsh** [comment]
Potential topics for this week (may depend on who can attend!):

- Any progress that Gary M. may have made on mapping out an ontology and how such an ontology might help us organize the remaining work
- Consider "What next?" with [Henry's diagram](https://modern-json-schema.com/analyzing-the-openapi-tooling-ecosystem) (but only if @handrews is available)
- Pick up the [security thread on Slack](https://open-api.slack.com/archives/C1137F8HF/p1728058609941179) that @kevinswiber kicked off

**Reply by handrews** [reply]
@earth2marsh please incorporate:

Get everyone on board with using the [color names provided with the diagram](https://modern-json-schema.com/analyzing-the-openapi-tooling-ecosystem#heading-color-coding-and-accessibility) (usually minus the "light" and "pale" qualifiers) to make the discussions accessible to those with impaired color vision (I did not pick these names, they [came with the color palette)](https://personal.sron.nl/~pault/#fig:scheme_light):

![colors](https://github.com/user-attachments/assets/b367961b-4af6-4a75-bfe9-d01c6adf77cb)

-----

Parsing libraries (lower region): Discuss the distinction between the DOM-like grey box and the very different pink ADA box (which still needn't be called "ADA", but it's not a "DOM" either)

-----

Also, folks asked for a left-right switch of OAD tools vs API tools, so here's that (but ***please***, read the [blog post](https://modern-json-schema.com/analyzing-the-openapi-tooling-ecosystem) if you have not already, as it is essential for understanding!):

![Idealized Architecture L-R Switched](https://github.com/user-attachments/assets/4416090a-c453-4d45-bb14-d5dd86290217)
